### PR TITLE
Properly move configuration files for locally installed wheels

### PR DESCRIPTION
**Pull request by ofek**
### Motivation

Implementation was incomplete

### CODE REVIEW

**Review by trishankatdatadog — APPROVED** [review]

**Comment by trishankatdatadog** [comment]
Doesn't look like changes security to the downloader

File: cmd/agent/app/integrations.go [hunk 1]
```diff
@@ -290,13 +292,38 @@ func install(cmd *cobra.Command, args []string) error {
...
+
+		// Install the wheel
+		if err := pip(append(pipArgs, wheelPath), os.Stdout, os.Stderr); err != nil {
+			return fmt.Errorf("error installing wheel %s: %v", wheelPath, err)
 		}
-		return pip(append(pipArgs, args[0]), os.Stdout, os.Stderr)
+
+		// Move configuration files
+		if err := moveConfigurationFilesOf(integration); err != nil {
+			fmt.Printf("Installed %s from %s", integration, wheelPath)
```
Comment (hush-hush) [inline cmd/agent/app/integrations.go]: why not use the `log` package here ?

**Review by hush-hush — APPROVED** [review]
LGTM, just a small question about the log package

File: cmd/agent/app/integrations.go (hunk 1 above)
Comment (ofek) [reply cmd/agent/app/integrations.go]: We want to display things in the console afaik

**Review by zippolyte — APPROVED** [review]
LGTM

**Pull request merged**